HpToolsLauncher/Launcher.cs

```diff
@@ -507,7 +513,7 @@ IAssetRunner CreateRunner(TestStorageType runType, JavaProperties ciParams)
                         mobileinfo = _ciParams["mobileinfo"];
                     }
 
-                    runner = new FileSystemTestsRunner(validTests, timeout, pollingInterval, perScenarioTimeOutMinutes, ignoreErrorStrings, jenkinsEnvVariables, mcConnectionInfo, mobileinfo);
+                    runner = new FileSystemTestsRunner(validTests, timeout, uftRunMode, pollingInterval, perScenarioTimeOutMinutes, ignoreErrorStrings, jenkinsEnvVariables, mcConnectionInfo, mobileinfo);
```

Comment:
its not a general method, although it looks like a general one.
it was only called in one place. adding an overloaded constructor will cause the previous constructor to have zero reference, IDE pops warning to tell you the constructor is not referenced anywhere.